### ISSUE TITLE
[feature request] blocking anonymous calls

### ISSUE THREAD

**Issue by gbdomubpkm** [jmlich/exphone#4]
First, thank you for having the great courage to start this call blocker with a user interface.

As other developers do, maybe you could open a thread on the forum, given the low performance of morph-browser at the moment (and github not very usable).

Anyway, I submit this request to block anonymous calls.

Best regards

**Comment by jmlich** [comment]
There are actually more cases:

First case is "not in address book". It is actually quite easy to implement.

Second case is with hidden phone number using "Calling Line Identification Restriction". I am not sure how used API behaves in this case. The right snippet of `journalctl --user -u exphoned` would help.

Feel free to send pull request.

**Comment by gbdomubpkm** [comment]
Yes, I'm talking about calls with hidden numbers. Unfortunately, I don't have the skills to make a pull request.

**Comment by jmlich** [comment]
Fixed in 10bc5defa410ecc6012915e3efda7f4267cd59c4
